db/schema.rb

```diff
@@ -54,6 +56,18 @@
     t.index ["shift_id"], name: "index_interims_on_shift_id"
   end
 
+  create_table "regional_managers", force: :cascade do |t|
```

Comment:
I removed this table in an earlier PR. Could you run `rails db:drop db:setup` and commit your changes?